site/releases/2023/2023-aug-15/highlights.qmd

```diff
@@ -13,36 +15,176 @@ This release includes a number of improvements for the developer experience when
 --->
 
 <!---John6797/sc 1980/metric test class dx improvements by @johnwalz97 in [#220](https://github.com/validmind/validmind-python/pull/220)--->
-- **Improved developer experience for the Metric test class**. We made a number of enhancements to the {{< var validmind.developer >}} to improve the experience for developers:
+#### Improved developer experience for the Metric test class
 
-   * **Better test organization**: Added a `metadata` and `tags` attribute to the `Test` base class for better categorization and filtering of tests.
-   * **Better filtering**: Added a new `task_type` argument to the `list_tests()` function to allow simple filtering by `task_type` set in a test.
-   * **Clearer naming**: Renamed `required_context` to `required_inputs` across the whole ValidMind SDK.
+We made a number of enhancements to the {{< var validmind.developer >}} to improve the experience for developers.
+
+##### Better test organization
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-80-ns}
+Added a `metadata` and `tags` attribute to the `Test` base class for better categorization and filtering of tests.
+:::
+
+::: {.w-20-ns}
+<form method="get" action="/validmind/validmind/vm_models.html#Test" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Test()
+</button>
+</form>
+
+:::
+
+::::
+
+##### Better filtering
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-80-ns}
+Added a new `task_type` argument to the `list_tests()` function to allow simple filtering by `task_type` set in a test.
+
+:::
+
+::: {.w-20-ns}
+<form method="get" action="/validmind/validmind/tests.html#list_tests" target="_blank">
```

Comment:
Interesting, they are fine locally, I checked them all. Just for this page? 🤔

Reply:
> Interesting, they are fine locally, I checked them all. Just for this page? 🤔

This is the only page I checked. The links look like they should work but the preview 404s.

Reply:
I had this moment of worry that the recent changes to how we handle the Python docs are implicated but they appear fine on staging: https://docs-staging.validmind.ai/validmind/validmind.html. Let me know if you want help troubleshooting?

Reply:
Hm, I think for some reason the demo-docs doesn't upload the Python docs: https://docs-demo.vm.validmind.ai/validmind/validmind.html

Just to be sure though, I'll just change the URLs to the full one for now.

Reply:
I think it's just a weird quirk of how the preview operates — it tries to send people to the top-level `validmind` folder instead of the one that's within the PR preview folder... (even though it doesn't do that for any of the other directories, maybe because the `validmind` exists in output only)?

Reply:
They should be good now but you'll need to empty cache/hard refresh/etc. hehe

Reply:
> Hm, I think for some reason the demo-docs doesn't upload the Python docs: https://docs-demo.vm.validmind.ai/validmind/validmind.html

The docs-demo site is quite old, I would not use it for reference. 

(Related, how do you feel about repurposing docs-demo for the chatbot? Once I figure it out with Cullen where the backend should live, we could use docs-demo for further exploration.)

Reply:
> Related, how do you feel about repurposing docs-demo for the chatbot? 

Go for it!

Reply:
> They should be good now but you'll need to empty cache/hard refresh/etc. hehe

They mostly work now but it should be OK. I think the difference is whether you are using 

`action="https://docs.validmind.ai/validmind/validmind/tests.html#describe_test`

or 

`action="/validmind/validmind.html#get_test_suite`

The latter, relative links seem to 404 but I expect they will work after you merge version. Imma approve. 🚀